### ISSUE TITLE
dpn feature request

### ISSUE THREAD

**Issue by qingzhouzhen** [apache/incubator-mxnet#8389]
I found [DPN:https://arxiv.org/abs/1707.01629](https://arxiv.org/abs/1707.01629) is implemented by MXNet:[https://github.com/cypw/DPNs](https://github.com/cypw/DPNs) but I did not found DPN in MXNet source code, May I reimplement it, verify it an contribute to MXNet? Or adjust it to MXNet?

**Comment by eric-haibin-lin** [comment]
If you can reproduce it in MXNet, contribution will be great!

**Comment by chinakook** [comment]
@qingzhouzhen, Can you reimplement [NASNet-A_Mobile_224 and NASNet-A_Large_331](https://arxiv.org/abs/1707.07012)? The former has 0.74 top-1 accuracy on ImageNet and has the same FLOPS with Mobilenet(0.708 top-1), the latter has **0.827** top-1 accuracy with only 9 GFlops(better than [DPN](https://github.com/cypw/DPNs) which is already implemented by MXNet). [Here](https://github.com/tensorflow/models/pull/2537/files) is some reference.

**Comment by qingzhouzhen** [comment]
OK, give me some time @chinakook

**Comment by qingzhouzhen** [comment]
I had reimplemented nasnet_large `https://github.com/qingzhouzhen/incubator-mxnet/blob/nasnet/python/mxnet/gluon/model_zoo/vision/nasnet.py` which refers to [Torch implement](https://github.com/Cadene/pretrained-models.pytorch/blob/master/pretrainedmodels/models/nasnet.py) and [Tensorflow implement](https://github.com/tensorflow/models/tree/master/research/slim/nets/nasnet), but I have no so many GPUs to train the model, any one could work on whis would be great.
